src/MoBi.Engine/Sbml/CompartmentImporter.cs

```diff
@@ -116,13 +118,19 @@ private void CreateMoleculeProperties(IContainer container)
       /// <summary>
       ///     Creates a Volume Parameter for a MoBi Container. 
       /// </summary>
-      private IEntity CreateVolumeParameter()
+      private IEntity CreateVolumeParameter(Compartment compartment)
       {
-         IFormula formula = ObjectBaseFactory.Create<ConstantFormula>().WithValue(1);
+         var volume = 1.0;
+         if (compartment.isSetVolume())
+            volume = compartment.getVolume();
+         else if (compartment.isSetSize())
+            volume = compartment.getSize();
+         var dimension = _dimensionFactory.Dimension(Constants.Dimension.VOLUME);
+         IFormula formula = _formulaFactory.ConstantFormula(volume, dimension);
```

Comment:
@Yuri05's point is valid. Is the `volume` value always in baseunit? or could the value be mL for instance? In this case, the value needs to be converted to baseunit. So it's not only a display issue but the value might be wrong